examples/Boltzmann_Wealth_Model/Readme.md

```diff
@@ -2,33 +2,33 @@
 
 ## Summary
 
-A simple model of agents exchanging wealth. All agents start with the same amount of money. Every step, each agent with one unit of money or more gives one unit of wealth to another random agent. This is the model described in the [Intro Tutorial](http://mesa.readthedocs.io/en/latest/intro-tutorial.html).
+A simple model of agents exchanging wealth. All agents start with the same amount of money. Every step, each agent with one unit of money or more gives one unit of wealth to another random agent. This is the model described in the [Intro Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html).
```

Comment:
Confirmed that this is the change done in https://github.com/projectmesa/mesa/commit/238c2c0bd6286970418bd1439bf0c136df665782.